lib/ember-orbit/store.js

```diff
@@ -1,379 +1,137 @@
-import Source from './source';
-import RecordArrayManager from './record-array-manager';
-import OCMemorySource from 'orbit-common/memory-source';
+import Schema from 'ember-orbit/schema';
+import Cache from 'ember-orbit/cache';
+import IdentityMap from 'ember-orbit/identity-map';
+import { operationType } from 'orbit-common/lib/operations';
 
 /**
  @module ember-orbit
  */
 
-var get = Ember.get;
+const get = Ember.get;
 
-var RSVP = Ember.RSVP;
-
-var PromiseArray = Ember.ArrayProxy.extend(Ember.PromiseProxyMixin);
-function promiseArray(promise, label) {
-  return PromiseArray.create({
-    promise: RSVP.Promise.cast(promise, label)
-  });
-}
-
-var Store = Source.extend({
-  orbitSourceClass: OCMemorySource,
+var Store = Ember.Object.extend({
+  orbitStore: Ember.inject.service('orbitStore'),
```

Comment:
`orbitStore` is the only dependency that needs to be passed in to create a Store. In the case of a Transaction it might be something like:

```
Transaction.create({
  baseStore: orbitStore,
  orbitStore: orbitStore.createTransaction(transactionOptions) // assuming we add orbitStore.createTransaction() in orbit.js
})
```